app/src/main/java/net/gsantner/markor/util/DocumentIO.java

```diff
@@ -117,7 +146,7 @@ public static synchronized Document loadDocument(Context context, Bundle argumen
         return document;
     }
 
-    public static synchronized boolean saveDocument(final Document document, final String text, final ShareUtil shareUtil) {
+    public static synchronized boolean saveDocument(final Document document, final String text, final ShareUtil shareUtil, Context context) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'saveDocument(Document, String, ShareUtil, Context)' has an NPath complexity of 244](https://app.codacy.com/manual/gsantner/markor/pullRequest?prid=5279643)